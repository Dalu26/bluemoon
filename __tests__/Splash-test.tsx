 import 'react-native';
 import React from 'react';
 import SplashScreen from '../src/screens/onboarding/splashscreen';
 
 // Note: test renderer must be required after react-native.
 import renderer from 'react-test-renderer';
//  jest.useFakeTimers()
 
 it('renders correctly', () => {
   const splash = renderer.create(<SplashScreen />)
   expect(splash).toMatchSnapshot();
 });
 