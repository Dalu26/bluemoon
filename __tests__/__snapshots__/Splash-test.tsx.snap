// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "rgba(98, 47, 181, 1)",
      "flex": 1,
      "paddingTop": 55.5,
    }
  }
>
  <BVLinearGradient
    colors={
      Array [
        4284624821,
        4279963446,
      ]
    }
    endPoint={
      Object {
        "x": 0.5,
        "y": 1,
      }
    }
    locations={null}
    startPoint={
      Object {
        "x": 0.5,
        "y": 0,
      }
    }
    style={
      Object {
        "flex": 1,
        "paddingBottom": 65,
        "paddingHorizontal": 48.5,
        "paddingTop": 55.5,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    />
    <View
      style={
        Object {
          "flex": 1.5,
          "justifyContent": "flex-end",
        }
      }
    >
      <View
        collapsable={false}
        forwardedRef={[Function]}
        style={
          Object {
            "transform": Array [
              Object {
                "translateY": 100,
              },
            ],
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#ffffff",
                "fontFamily": "Poppins-Regular",
                "fontSize": 26,
              },
              Object {
                "fontFamily": "Poppins-Bold",
                "fontSize": 56,
                "fontWeight": "600",
                "lineHeight": 76,
              },
            ]
          }
        >
          Bluemoon Solutions
        </Text>
        <Text
          style={
            Array [
              Object {
                "color": "#ffffff",
                "fontFamily": "Poppins-Regular",
                "fontSize": 26,
              },
              Object {
                "fontSize": 28,
                "marginBottom": 110.5,
                "marginTop": 14,
              },
            ]
          }
        >
          Check and update your inventory with ease
        </Text>
      </View>
      <View
        collapsable={false}
        forwardedRef={[Function]}
        style={
          Object {
            "opacity": 0,
            "transform": Array [
              Object {
                "scale": 1,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "width": "100%",
            }
          }
        >
          <View
            accessibilityState={
              Object {
                "disabled": false,
              }
            }
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#7F4CD2",
                "borderRadius": 39.5,
                "display": "flex",
                "height": 155,
                "justifyContent": "center",
                "opacity": 1,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "display": "flex",
                  "flexDirection": "row",
                  "justifyContent": "center",
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#ffffff",
                      "fontFamily": "Poppins-Regular",
                      "fontSize": 26,
                    },
                    Array [
                      Object {
                        "fontFamily": "Poppins-Medium",
                        "fontSize": 50.5,
                        "fontWeight": "600",
                        "lineHeight": 76,
                      },
                      undefined,
                      Object {
                        "color": "#ffffff",
                      },
                    ],
                  ]
                }
              >
                Get started
              </Text>
            </View>
          </View>
        </View>
      </View>
    </View>
  </BVLinearGradient>
</RCTSafeAreaView>
`;
